examples/express-es6/package.json

```diff
@@ -13,14 +13,14 @@
   "author": "Sami Koskimäki",
   "license": "MIT",
   "dependencies": {
-    "axios": "^0.17.0",
+    "axios": "^0.17.1",
     "body-parser": "^1.18.2",
     "express": "^4.16.2",
-    "express-promise-router": "^2.0.0",
-    "knex": "^0.13.0",
+    "express-promise-router": "^3.0.1",
+    "knex": "^0.14.2",
```

Comment:
knex 0.14.2 has known problems with memory leaks, I wouldn't recommend updating to it before 0.14.3 is out.

Reply:
👍